### ISSUE TITLE
r5_dockerize: /home not created for new clusters

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#445]
**Version:** r5_dockerize 38875bd
amaster

```
Last login: Fri Sep 18 13:46:31 2015 from login-new.vc-new
[root@login ~]# cd /home
[root@login home]# mount -a
mount.nfs: mounting controller:/home/vc-new failed, reason given by server: No such file or directory
```
```
[root@controller ~(keystone_new-user)]# ls /home
trinity  vc-a
```

**Comment by abhimukh** [comment]
@themisa The commit message had a typo. Could you reproduce this issue?

**Comment by themisa** [comment]
Fixed in **Version:** r6 ea67ae8
```
[root@controller(unknown*) ~]# ls /home
trinity  vc-a  vc-new

[root@login ~]# df
Filesystem              1K-blocks     Used Available Use% Mounted on
/dev/vda1                20953560  1526840  19426720   8% /
devtmpfs                  1007488        0   1007488   0% /dev
tmpfs                     1024704        0   1024704   0% /dev/shm
tmpfs                     1024704    16672   1008032   2% /run
tmpfs                     1024704        0   1024704   0% /sys/fs/cgroup
10.141.255.254:/trinity 114480528 33362280  81118248  30% /trinity
controller:/home/vc-new 114480528 33362280  81118248  30% /home

```
